backend/lucene/src/main/java/org/hibernate/search/backend/lucene/index/impl/LuceneIndexManagerImpl.java

```diff
@@ -207,7 +208,7 @@ public long computeSizeInBytes() {
 
 	@Override
 	public CompletableFuture<Long> computeSizeInBytesAsync() {
-		return schemaManager.computeSizeInBytes();
+		return schemaManager.computeSizeInBytes( OperationSubmitter.REJECTED_EXECUTION_EXCEPTION );
 	}
 
```

Comment:
That's great, thanks.

Now, I'm sorry I took all that time to realize, but it just occurred to me that we're hardcoding the submitter in a *backend*... which might be a problem for Infinispan. That's fine for APIs, though.

So... could you please add a new method `CompletableFuture<Long> computeSizeInBytesAsync(OperationSubmitter)` to `LuceneIndexManager`, so that Infinispan can pick the submitter at will? Never too careful with these things...

If you adapt `computeSizeInBytesAsync()` (no-arg) to call you new method, you probably don't need to add a test (it'll be covered by existing tests).